File: Euclid/Messages/Incoming/GETCREDITS.cs
using Euclid.Game;
using Euclid.Messages.Outgoing;
using Euclid.Network.Streams;

namespace Euclid.Messages.Incoming
{
    class GETCREDITS : IMessageEvent
    {
        public void Handle(Player player, Request request)
        {
            player.Send(new WALLETBALANCE(999999));
        }
    }
}

File: euclid/src/main/java/euclid/messages/incoming/GETCREDITS.java
package euclid.messages.incoming;

import euclid.game.Player;
import euclid.messages.IMessageEvent;
import euclid.messages.outgoing.WALLETBALANCE;
import euclid.network.streams.Request;

public class GETCREDITS implements IMessageEvent {

    @Override
    public void handle(Player player, Request request) {
        player.send(new WALLETBALANCE(999999));
    }
}
